tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -235,4 +237,231 @@ describe("E2E Testing for taqueria flextesa plugin",  () => {
         }
     })
 
-});
\ No newline at end of file
+});
+
+describe("E2E Testing for taqueria typechecker and simulator tasks of the flextesa plugin", () => {
+
+    beforeAll(async () => {
+        dockerName = "local"
+        await generateTestProject(taqueriaProjectPath, ["flextesa"]);
+        await exec(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`})
+    })
+
+    test('Verify that taqueria typechecker task can typecheck one contract under contracts folder', async () => {
+        try {
+            // 1. Copy contract from data folder to taqueria project folder
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts`);
```

Comment:
We have had problems with mixing sync and async functions in the past. Can you try to use async functions when you can?

Reply:
You mean use `await exec` instead of `execSync`?

Reply:
Will reflect in the next commit